### PR TITLE
Feature/weather icon

### PR DESCRIPTION
**Pull request by rchl**
Implement support for specifying image URL for weather icon (#78)

### CODE REVIEW

**Comment by rchl** [comment]
Note that I've used template literals in one place so this can break older browsers (IE11). Not sure if the aim is to support that one.

**Comment by Evgeny-** [comment]
Doesn't seem like "url:" prefix is a good solution

**Comment by rchl** [comment]
Other options I can think of is to have either completely separate set of properties (iconUrl, iconUrlMap) or one extra property (e.g. iconAsUrl: true/false) which would change behaviour of icon/iconMap.

(Also, URL relative to server URL should probably be handled too)

**Comment by Evgeny-** [comment]
I think `icon` and `iconUrl` would be ok. iconMap we deprecate and move them to `icons`. `iconUrl` would have greater priority.

File: scripts/directives.js [hunk 1]
```diff
@@ -257,3 +257,46 @@ App.directive('iframeTile', ['$interval', function ($interval) {
...
+      `,
+      link: function ($scope, $el, attrs) {
+         $scope.iconClass = '';
+         $scope.iconStyle = {};
+
+         var updateIcon = function () {
+            var iconValue = $scope.getWeatherIcon({
+               item: $scope.item,
+               entity: $scope.entity,
+            });
```
Comment (Evgeny-) [inline scripts/directives.js]: Is it a mistake? Calling function with object as argument?

File: scripts/directives.js [hunk 2]
```diff
@@ -257,3 +257,46 @@ App.directive('iframeTile', ['$interval', function ($interval) {
       }
    }
 }]);
+
+App.directive('weatherIcon', [function () {
```
Comment (Evgeny-) [inline scripts/directives.js]: Not sure if we really need directive

File: scripts/directives.js (hunk 1 above)
Comment (rchl) [reply scripts/directives.js]: In angularjs, when function is bound to a scope, it has to be called that way. The object must have keys that match the signature at the bound-site.

Comment (Evgeny-) [reply scripts/directives.js]: Oh, it's isolated (&). Well anyway, i'm think we don't really need directive here.

File: scripts/directives.js (hunk 2 above)
Comment (Evgeny-) [reply scripts/directives.js]: We can just separate elements, i'll try to do it.

**Comment by Evgeny-** [comment]
Somehow implemented

**Pull request closed without merging**